deploy/nfpm/README.md

```diff
@@ -28,13 +34,13 @@ Download the latest package from https://github.com/hoprnet/hoprnet/releases/lat
 
 ```bash
 arch=$(uname -m); [[ "$arch" == "arm64" ]] && arch="aarch64"
-curl -s -L -o hoprd-${arch}-linux.rpm https://github.com/hoprnet/hoprnet/releases/download/latest/hoprd-${arch}-linux.rpm
-# You can get Safe address and the Module addres from https://hub.hoprnet.org
+curl -s -L -o hoprd_${arch}.rpm https://github.com/hoprnet/hoprnet/releases/download/latest/hoprd_${arch}.rpm
```

Comment:
I prefer dash instead of underscores in binary names. Why the switch?

Reply:
In Debian, the standard and compliant package naming convention follows the format:

<name>_<version>-<revision>_<architecture>.deb